__tests__/main.test.ts

```diff
@@ -3,76 +3,172 @@ import * as process from 'process'
 import * as cp from 'child_process'
 import * as path from 'path'
 import * as rimraf from 'rimraf'
-import { downloadLatest, install } from '../src/restore'
-import { getInstallDir, getAccessToken } from '../src/lib'
-
-// import { downloadLatest } from '../lib/restore'
+import * as fs from 'fs'
+import { getInstallDir } from '../src/lib'
 
 jest.setTimeout(40000)
 
 const cachePath = path.join(__dirname, 'runner', 'CACHE')
 const tempPath = path.join(__dirname, 'runner', 'TEMP')
 const ghWorkspace = path.join(__dirname, 'runner', 'WORKSPACE')
+const buildcacheDir = path.join(ghWorkspace, '.buildcache', 'cache')
 
 // Set temp and tool directories before importing (used to set global state)
 process.env['RUNNER_TEMP'] = tempPath
 process.env['RUNNER_TOOL_CACHE'] = cachePath
 process.env['GITHUB_WORKSPACE'] = ghWorkspace
+process.env['BUILDCACHE_DIR'] = buildcacheDir
```

Comment:
Set `BUILDCACHE_DIR` to a folder inside the workspace, to automatially wipe the cache between tests